spec/features/idv/in_person_threatmetrix_spec.rb

```diff
@@ -0,0 +1,299 @@
+require 'rails_helper'
+require 'action_account'
+require 'axe-rspec'
+
+RSpec.describe 'In Person Proofing Threatmetrix', js: true, allowed_extra_analytics: [:*] do
+  include InPersonHelper
+
+  let(:sp) { :oidc }
+  let(:user) { user_with_2fa }
+  let(:enrollment_code) do
+    JSON.parse(
+      UspsInPersonProofing::Mock::Fixtures.request_enroll_response,
+    )['enrollmentCode']
+  end
+  let(:stdout) { StringIO.new }
+  let(:stderr) { StringIO.new }
+  let(:argv) { ['review-pass', user.uuid, '--reason', 'INV1234'] }
+  let(:action_account) { ActionAccount.new(argv:, stdout:, stderr:) }
+  let(:review_pass) { ActionAccount::ReviewPass.new }
+  let(:review_reject) { ActionAccount::ReviewReject.new }
+  let(:include_missing) { true }
+  let(:config) { ScriptBase::Config.new(include_missing:, reason: 'INV1234') }
+  let(:enrollment) { InPersonEnrollment.last }
+  let(:profile) { enrollment.profile }
+
+  before do
+    allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+    allow(IdentityConfig.store).to receive(:in_person_proofing_enforce_tmx).and_return(true)
+    ServiceProvider.find_by(issuer: service_provider_issuer(sp)).
+      update(in_person_proofing_enabled: true)
+  end
+
+  def deactivate_profile_update_enrollment(status:)
+    # set fraud review to pending:
+    profile.deactivate_for_fraud_review
+    # update the enrollment:
+    enrollment.update(
+      status: status,
+      proofed_at: Time.zone.now,
```

Comment:
The `proofed_at` and `status_check_completed_at` values are never checked. So I suppose we could remove them, but I wanted to more closely emulate how the enrollment gets updated.